client/deis.py

```diff
@@ -316,13 +316,14 @@ def readable_datetime(datetime_str):
             return 'Just now'
         else:
             return "{}{}ago".format(hour_str, min_str)
+
     # if it happened yesterday, say "yesterday at 3:23 pm"
-    yesterday = now + relativedelta.relativedelta(days=-1)
-    if delta.days <= 2 and dt.day == yesterday.day:
```

Comment:
relativedelta obj is not acting the way this code expects:
```python
In [1]: from dateutil import tz

In [2]: from dateutil import parser

In [3]: from datetime import datetime

In [4]: from dateutil import relativedelta

In [5]: 

In [5]: now = datetime.now(tz.tzlocal())

In [6]: datetime_str = '2015-02-04T09:14:34UTC'

In [7]: dt = parser.parse(datetime_str).astimezone(tz.tzlocal())

In [8]: r = relativedelta.relativedelta(now, dt)

In [9]: r
Out[9]: relativedelta(months=+1, days=+1, hours=+2, minutes=+55, seconds=+55, microseconds=+527323)

In [10]: r.days
Out[10]: 1
```

`r.days` is `1` and not the actual number of days between the two dates.